HW03/roman_nasyrov/roman_nasyrov_hw_03_t_02.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# declaring class
+class Homework3
+  REGEX = %r{^([\d+.\d+]+).*(?<=\[)(.*)(?=\]).*(?<=\s)((?:/\w+)*?)(?=\s).*$}.freeze
```

Comment:
There's only one rubocop error, maybe you didn't save the changes? 🤔 
```suggestion
  REGEX = %r{^([\d+.\d+]+).*(?<=\[)(.*)(?=\]).*(?<=\s)((?:/\w+)*?)(?=\s).*$}
```